app/views/links/listing_property.erb

```diff
@@ -0,0 +1,17 @@
+<h2> Property Information </h2>
+
+<form action='/property' method='post'>
+  <label for='name'>
+    Name: <input name='name' type='text'>
+  </label>
+  <label for='description'>
+    Description: <input name='description' type='text'>
```

Comment:
Overall code looks good. However, there is an inconsistency with the property model and tests, as the user can input all information from the user story, but we currently do not have this tested or stored.